.eslintrc.js

```diff
@@ -1,20 +0,0 @@
-module.exports = {
-  extends: ["eslint:recommended", "plugin:@typescript-eslint/recommended", "plugin:@typescript-eslint/recommended-requiring-type-checking", "plugin:@typescript-eslint/recommended-requiring-type-checking"
-],
-  parser: "@typescript-eslint/parser",
-  plugins: ["@typescript-eslint"],
-  root: true,   
-  parserOptions: {
-    sourceType: "module",
-    tsconfigRootDir: __dirname,
-    project: ["./tsconfig.json"],
-  },
-  rules: {
-    "@typescript-eslint/no-floating-promises": "error",
-    "@typescript-eslint/no-unsafe-member-access": "off",
-    "@typescript-eslint/no-explicit-any": "off",
-    "@typescript-eslint/no-unsafe-call": "off",
-    "@typescript-eslint/no-unsafe-assignment": "off",
-    "@typescript-eslint/no-inferrable-types": "off",
```

Comment:
those rules are missing in the updated .mjs file below.

Reply:
same for `plugin:@typescript-eslint/recommended-requiring-type-checking`

Reply:
added into mjs file